client/components/App.jsx

```diff
@@ -66,7 +73,7 @@ const CustomerQuestions = () => {
 
 
   const getCustomerQuestionsData = (productId) => {
-    axios.get(`http://ec2-3-22-93-125.us-east-2.compute.amazonaws.com:4001/customer-questions/${productId}`)
+    axios.get(`http://localhost:4001/customer-questions/${productId}`)
```

Comment:
Any reason for switching to making a request to your local host?

Reply:
Sorry, this was committed by accident. I haven't set up environment variables and I am still working on features!